Yamb/YambDataSource.swift

```diff
@@ -79,11 +79,17 @@ class YambDataSource {
         return [0:topFields, 1:middleFields, 2:bottomFields]
     }()
     
-    var totalScore: Int {
+    lazy var totalScore: Int = {
```

Comment:
lazy var initializers are only called once. this should be a computed property. it is enough to say 
`var totalScore: Int = { getTotalScore() }`

Reply:
<img width="330" alt="Screenshot 2022-10-20 at 12 05 03" src="https://user-images.githubusercontent.com/94114907/196920041-04c58e25-1545-445f-a9c0-b6e98899983b.png">